ui/app/AppLayouts/Onboarding2/pages/LoginScreen.qml

```diff
@@ -218,7 +218,7 @@ OnboardingPage {
                     }
 
                     Qt.callLater(() => {
-                        if (!isBiometricsLogin)
+                        if (!root || !root.isBiometricsLogin)
```

Comment:
How could it happen there's no `root`? :thinking:

Reply:
That's good question indeed.

There is quite strange behavior on Qt 6 - there are 2 instances of login screen created, the first one is immediately destroyed, and the second one seems to be used. As the first one has keyUid set, the callLater is triggered, but object itself is immediately destroyed. So when the lambda is executed it's already over... Needs some deeper investigation probably. Maybe next Qt 6 bug? :thinking:

Reply:
Yeah... looks like you found a new one :laughing:

Reply:
https://github.com/status-im/status-desktop/issues/17453